akka-docs/src/main/paradox/scala/guide/quickstart.md

```diff
@@ -1,136 +1,26 @@
 # Quickstart
```

Comment:
One thing I'm still not completely satisfied with is that I would like to have a "Try Akka" landing page, and this is kind of more part of the Getting Started Guide than being an entry page. The first choice must also be Scala / Java and that would be slightly strange to have here.

I suggest that we have this page like this in the documentation, but also have a very similar page at the website, Try Akka, that we can use as the Quickstart landing page, and also include the choice Java / Scala. Similar to https://www.lagomframework.com/get-started.html

WDYT?